### PR TITLE
Feature/cast of movie

### PR DESCRIPTION
**Pull request by leomoraiscam**
This PR implemented the functionality to create casts for films and also to list the cast of films

### CODE REVIEW

File: src/services/casts/create.service.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { StatusCodes } = require('http-status-codes');
+const { castsRepository } = require('../../repositories');
+const { ApplicationError } = require('../../utils');
+const { messages } = require('../../helpers');
+
+module.exports = {
+  create: async (params) => {
+    const existCast = await castsRepository.get({
+      name: params.name,
```
Comment (rafaelrodriguesioasys) [inline src/services/casts/create.service.js]: Sugiro acrescentar o operado ilike

File: src/config/database/migrations/20210201180941-CreateTableCast.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      },
+      character: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      avatar: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      id_movie: {
```
Comment (rafaelrodriguesioasys) [inline src/config/database/migrations/20210201180941-CreateTableCast.js]: dúvida: Qual o padrão do projeto:
id_movie ou movie_id?

Comment (leomoraiscam) [reply src/config/database/migrations/20210201180941-CreateTableCast.js]: movie_id

File: src/services/casts/create.service.js (hunk 1 above)
Comment (leomoraiscam) [reply src/services/casts/create.service.js]: feito!

**Review by rafaelrodriguesioasys — APPROVED** [review]

**Pull request merged**